basics/supporting.md

```diff
@@ -111,12 +111,22 @@ $ docker run -d -p 9200:9200 -p 5601:5601 nshou/elasticsearch-kibana
 Then you can connect to Elasticsearch by localhost:9200 and its Kibana front-end by localhost:5601. See [details about these Docker images in DockerHub](https://hub.docker.com/r/nshou/elasticsearch-kibana/)
 
 
-### Installing MariaDB
+### Installing MariaDB / MySQL
 
 If you are going to use SortingHat, you will need a database. Currently, MySQL-like databases are supported. In our case, we will use MariaDB. Installing it in Debian is easy:
 
 ```bash
 $ sudo apt-get install mariadb-server
 ```
 
-That's it, that's all.
\ No newline at end of file
+That's it, that's all.
+
+If you're not lucky enough to work on a Debian derivatave, check for installation instructions for MariaDB or MySQL in your system. Or give a try to a preinstalled Docker container. For example, you can run MySQL from a container as:
+
+```
+docker run --env MYSQL_ALLOW_EMPTY_PASSWORD=1 --env MYSQL_ROOT_PASSWORD="" -p 3306:3306 -d mysql/mysql-server:latest
```

Comment:
```suggestion
docker run -d -p 3306:3306 -e MYSQL_ALLOW_EMPTY_PASSWORD=yes mysql
```

With the above suggested change, you can also remove everything about creating a new root user, connecting to the running docker, etc - much simpler! Here are the two docker commands I use to spin up the various infrastructural bits for running tests for grimoirelab programs:

```
docker run -d -p 3306:3306 -e MYSQL_ALLOW_EMPTY_PASSWORD=yes mysql
docker run -d -p 9200:9200 -p 5601:5601 nshou/elasticsearch-kibana
```